backend/src/test/java/io/debezium/configserver/util/Infrastructure.java

```diff
@@ -96,7 +96,10 @@ public enum DATABASE {
                 .withNetwork(NETWORK)
                 .withKafka(KAFKA_CONTAINER.getNetwork(), KAFKA_HOSTNAME + ":9092")
                 .withLogConsumer(new Slf4jLogConsumer(LOGGER))
-                .dependsOn(KAFKA_CONTAINER);
+                .dependsOn(KAFKA_CONTAINER)
+                .withStartupTimeout(Duration.ofSeconds(120))
+                .withEnv("ENABLE_JOLOKIA", "true")
+                .withExposedPorts(8083, 8778);
```

Comment:
@Naros  Shouldn't this be added to a DebeziumContainer builder in the core repo? something like `#enableJolokia()` there?

Reply:
That seems reasonable since Jolokia is baked in.